array-destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,22 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+let { firstName, lastName, house, pet} = hogwarts;
```

Comment:
We don't need this information in the global scope, so there's no need to destructure here 🙂